### PR TITLE
Unite the word used in traditional_chinese.txt

### PR DESCRIPTION
**Pull request by By622**
## Checklist

- [ ] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

**Review by def- — APPROVED** [review]

**Comment by ASKLL-STAR** [comment]
还是没严格统一"檢查本**機**控制台以取得詳情":D

File: data/languages/traditional_chinese.txt [hunk 1]
```diff
@@ -2047,19 +2047,19 @@ Error loading touch controls
...
-== 無法從文件加載觸控設置。查看本地控制台以獲取詳情。
+== 無法從文件加載觸控設置。檢查本地控制台以獲取詳情。
 
 Could not load default touch controls from file. See local console for details.
-== 無法從文件加載默認觸控設置。查看本地控制台以獲取詳情。
+== 無法從文件加載默認觸控設置。檢查本地控制台以獲取詳情。
 
 Could not load touch controls from clipboard. See local console for details.
-== 無法從剪貼板加載觸控設置。查看本地控制台以獲取詳情。
+== 無法從剪貼板加載觸控設置。檢查本地控制台以獲取詳情。
```
Comment (ASKLL-STAR) [inline data/languages/traditional_chinese.txt:2056]:
```suggestion
== 無法從剪貼板加載觸控設置。檢查本機控制台以取得詳情。
```

File: data/languages/traditional_chinese.txt [hunk 2]
```diff
@@ -2047,19 +2047,19 @@ Error loading touch controls
 == 加載觸控設置錯誤
 
 Could not load touch controls from file. See local console for details.
-== 無法從文件加載觸控設置。查看本地控制台以獲取詳情。
+== 無法從文件加載觸控設置。檢查本地控制台以獲取詳情。
 
 Could not load default touch controls from file. See local console for details.
-== 無法從文件加載默認觸控設置。查看本地控制台以獲取詳情。
+== 無法從文件加載默認觸控設置。檢查本地控制台以獲取詳情。
```
Comment (ASKLL-STAR) [inline data/languages/traditional_chinese.txt:2053]:
```suggestion
== 無法從文件加載默認觸控設置。檢查本機控制台以取得詳情。
```

File: data/languages/traditional_chinese.txt [hunk 3]
```diff
@@ -2047,19 +2047,19 @@ Error loading touch controls
 == 加載觸控設置錯誤
 
 Could not load touch controls from file. See local console for details.
-== 無法從文件加載觸控設置。查看本地控制台以獲取詳情。
+== 無法從文件加載觸控設置。檢查本地控制台以獲取詳情。
```
Comment (ASKLL-STAR) [inline data/languages/traditional_chinese.txt:2050]:
```suggestion
== 無法從文件加載觸控設置。檢查本機控制台以取得詳情。
```

**Pull request merged**